Rakefile

```diff
@@ -63,3 +79,20 @@ def transform_for_es8!(template)
 
   nil
 end
+
+# destructively transforms an ES8-style legacy index_template into an ES7-compatible legacy template.
+def transform_for_es7!(template)
```

Comment:
As long as ECS generates ES7-style legacy templates on the 8.0 branch, this will be orphan, unreachable code.